nav2_bt_navigator/behavior_trees/navigate_with_immediate_replanning.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+  This Behavior Tree replans the global path at 0.5hz OR 2m distance OR when a new goal is available
+-->
+
+<root main_tree_to_execute="MainTree">
```

Comment:
Awesome, we have open questions on using both the time and distance, but we should also update existing BTs that do one or the other to use the new goal thing as well since that's obviously superior.